ale_linters/haskell/ghc.vim

```diff
@@ -7,69 +7,18 @@ endif
 
 let g:loaded_ale_linters_haskell_ghc = 1
 
-function! ale_linters#haskell#ghc#Handle(buffer, lines)
```

Comment:
Test the GHC handler again. It did some funky things. I might have fixed this now with changes I have made since I first wrote it to the `stdin-wrapper` script. It might have been an IFS bug.